ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -1,113 +1,132 @@
+    <ol>
+      <li>fill (the color)</li>
+      <li>size (at this time, only 16 (default) and 24 are supported)</li>
+      <li>display (inline-block or block)</li>
+      <li>additional CSS classes</li>
+    </ol>
+  </p>
+</div>
 
-<h3 class="ds-h3" id="accessibility"><a href="#accessibility">&sect;</a> Accessibility</h3>
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+  <h4 class="ds-h4">Examples</h4>
+  <p id="example-fill">
+    <strong><a href="#example-fill" class="ds-a" rel="external">&sect;</a>Fill:</strong> To customize the fill attribute, set the <kbd class="ds-kbd">@color</kbd> value (multiple supported ways). 
+    The recommended approach to ensure consistency is to use one of the pre-defined variables:
+    <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;zap&quot; @color=&quot;var(--brand)&quot; />" />
+  </p>
+  <p>
+    Other accepted values include named colors and color values themselves.
+  <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;zap&quot; @color=&quot;rebeccapurple&quot; />" />
+  <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;zap&quot; @color=&quot;rgb(46, 113, 229)&quot; />" />
+  </p>
 
-<p class="ds-p">
-Accessibility (a11y) support for SVGs is inconsistent across browsers and assistive technology. Currently, best practice is to set the <kbd class="ds-kbd">`aria-hidden`</kbd> attribute to `false` on the SVG itself.
-This means that the icon (both the singular icon and the icon component) will need to be used <em class="ds-em">in context</em>.
-The icons themselves are for presentation purposes only and should never be used on their own.
-</p>
+  <p id="example-size">
+    <strong><a href="#example-size" class="ds-a" rel="external">&sect;</a>Size:</strong> To use the 24x24 (px) icon size, set the <kbd class="ds-kbd">@size</kbd> value:
+    <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;zap&quot; @size=&quot;24&quot; />" />
+  </p>
+  <p id="example-styles">
+    <strong><a href="#example-styles" class="ds-a" rel="external">&sect;</a>CSS Classes:</strong> To append additional classes to the component, add <kbd class="ds-kbd">class</kbd> with value(s):
+    <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;triangle-fill&quot; class=&quot;ds-rotate-90&quot; />" />
+  </p>
+  <p id="example-display">
+    <strong><a href="#example-display" class="ds-a" rel="external">&sect;</a>CSS Display:</strong> To change the default display of <i>inline-block</i> to <i>block</i>, set <kbd class="ds-kbd">@isInlineBlock</kbd> to false:
+    <CodeBlock @language="markup" @code="&lt;FlightIcon @name=&quot;triangle-fill&quot; @isInlineBlock=&lbrace;&lbrace;false&rbrace;&rbrace; />" />
+  </p>
+</div>
 
-<h4 class="ds-h4">Example</h4>
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+  <h2 class="ds-h2" id="use-other"><a href="#use-other" class="ds-a" rel="external">&sect;</a> Using the flight-icon package in other ways</h2>
```

Comment:
reminder: later when the new packages will be released, we'll have to update this section with a more detailed explanation of how to use them (or alternatively, point to the README in the repo if we want to favour the usage as Ember Addon)

Reply:
Tracking the issue here: https://github.com/hashicorp/flight/issues/204